### DISCUSSION TITLE
Controller not working, steam deck

### DISCUSSION THREAD

**Discussion by Jippycats** [LizardByte/community#222] [Sunshine] [answered]
When streaming to a steam deck via windows pc using sushine + moonlight, I'm getting working picture and sound, but any controller inputs aren't working, and yes, I have vigem installed. Any ideas on how to fix are greatly appreciated!

**Comment by ReenigneArcher** [comment, marked as answer]
Did you restart the host after installing vigembus?

**Reply by Jippycats** [reply]
Just did, and fixed the issue, thank you!
